test/API.test.js

```diff
@@ -1,5 +1,6 @@
 var middleware = require("../middleware");
 var should = require("should");
+require("mocha-sinon");
```

Comment:
please move this to line 1 and add a line break between it and `var middleware...`